api/v3/PaymentProcessor.php

```diff
@@ -186,6 +201,11 @@ function _civicrm_api3_payment_processor_pay_spec(&$params) {
     'title' => ts('Invoice ID'),
     'type' => CRM_Utils_Type::T_STRING,
   ];
+  $params['currency'] = [
+    'title' => ts('Curency'),
+    'type' => CRM_Utils_Type::T_STRING,
+    'api.default' => Civi::settings()->get('defaultCurrency'),
+  ];
```

Comment:
Can we add the description parameter here with an example - as the input format has to be of the three letter kind - is it iso4217? https://www.xe.com/pt/iso4217.php